mlflow/tag.go

```diff
@@ -0,0 +1,7 @@
+package mlflow
+
+// Tag ...
+type Tag struct {
```

Comment:
what is the justification for a file with 7 lines and just a single entity? please move it to resource_experiment.go.

Reply:
done